lib/src/models/multireddit.dart

```diff
@@ -241,8 +244,23 @@ class Multireddit extends RedditBase with RedditBaseInitializedMixin {
           .replaceAll(_multiredditRegExp, name)
           .replaceAll(_userRegExp, user);
 
+  // Converts input text into camelCase.
+  static String _convertToCamelCase(String str) {
+    if (str == null) {
+      return null;
+    }
+    final RegExp _invalidRegExp = new RegExp(r'(\s|\W|_)+');
+    final splitStrList = str.split(_invalidRegExp);
+    // https://giphy.com/gifs/doctor-who-david-tennant-XTJACShBsxPws
+    splitStrList[0] += splitStrList.sublist(1)?.fold(
```

Comment:
Might be more readable to create another variable instead of building the final result in the first element of this list.